build.sbt

```diff
@@ -4,7 +4,7 @@ organization := "uk.ac.wellcome"
 
 name := "sierra-streams-source"
 
-version := "0.1"
+version := "0.3"
```

Comment:
Do you need me to release it?

Reply:
Yes, let’s!

I need a Travis account with keys/credentials, and a list of commands you run to release it.